.tekton/update-dependencies.yaml

```diff
@@ -44,12 +44,34 @@ spec:
             value: "origin/$(params.source_branch)"
           - name: refspec
             value: "refs/heads/$(params.source_branch)"
-      - name: update-dockerfiles
+      - name: git-setup
         runAfter:
           - git-clone
         workspaces:
           - name: source
             workspace: source
+        params:
+          - name: source_branch
+            value: $(params.source_branch)
+        taskSpec:
+          params:
+            - name: source_branch
+          steps:
+            - name: setup-local-repository
+              image: quay.io/redhat-pipeline-service/dependencies-update:$(params.source_branch)
+              imagePullPolicy: Always
+              workingDir: $(workspaces.source.path)
+              command:
+                - $(workspaces.source.path)/developer/images/dependencies-update/hack/bin/setup-local-repository.sh
```

Comment:
Because this step defines property `workingDir` ,it doesn't need to specify the root directory  `$(workspaces.source.path)` again in the command

Reply:
fixed